pkg/lvgl/doc.txt

```diff
@@ -26,4 +26,18 @@ Example of command line for changing the max activity period to 5s:
 CFLAGS=-DCONFIG_LVGL_ACTIVITY_PERIOD=5000 make -C tests/pkg_lvgl
 ```
 
+### SDL Requirements
+
+To run lvgl inside a simulator (native), SDL is required. Install instructions
+can be found at https://docs.lvgl.io/latest/en/html/get-started/pc-simulator.html#install-sdl-2.
```

Comment:
Would be worth adding a note that on Debian / Ubuntu one can just install the `libsdl2-dev:i386` package.